src/components/Workspace.js

```diff
@@ -39,6 +49,23 @@ const mapDispatchToProps = (dispatch, { cookies }) => ({
   ).catch(checkAuthError(dispatch)),
 });
 
+Blockly.Blocks.button_press = ButtonPress.definition;
```

Comment:
Is this possible because of the new way Blockly is built/packaged, or could have have done this all along?